code/game/atoms_movable.dm

```diff
@@ -69,6 +69,9 @@
 	///Highest-intensity light affecting us, which determines our visibility.
 	var/affecting_dynamic_lumi = 0
 
+	/// Whether this atom should have its dir automatically changed when it moves. Setting this to FALSE allows for things such as directional windows to retain dir on moving without snowflake code all of the place.
+	var/set_dir_on_move = TRUE
```

Comment:
minor point that i guess might be out of scope but I feel like movement related bools should really just be condensed into a bitfield to save memory